AnimatedWordCloud/Animator/AllocationCalculator/StaticAllocationCalculator/StaticAllocationCalculator.py

```diff
@@ -65,7 +65,10 @@ def allocate(
     # calculate allocation by selected strategy
     if config.allocation_strategy == "magnetic":
         allocator = MagneticAllocation(
-            config.image_width, config.image_height, config.image_division
+            config.image_width,
+            config.image_height,
+            config.image_division,
+            config.verbosity,
         )
         return allocator.allocate(words, allocation_before)
     else:
```

Comment:
function estimate_text_size's output type is weird. (int, int)->tuple[int, int]?

Reply:
damm, fixed